ui/raidboss/data/06-ew/raid/p3s.ts

```diff
@@ -320,6 +334,89 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P3S Sunbird Tether',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0039' }),
+      promise: async (data) => {
+        const callData = await callOverlayHandler({
+          call: 'getCombatants',
+        });
+        if (!callData || !callData.combatants || !callData.combatants.length) {
+          console.error('SunbirdTether: failed to get combatants: ${JSON.stringify(callData)}');
+          return;
+        }
+        const sunbirds = callData.combatants.filter((c) => c.BNpcID === 13635);
+        if (sunbirds.length !== 4) {
+          console.error('SunbirdTether: There is not exactly four Sunbirds?!?: ${JSON.stringify(sunbirds)}');
+          return;
+        }
+        data.sunbirds = sunbirds;
+      },
+      alertText: (data, matches, output) => {
+        ++data.sunbirdTetherCounter;
+        if (data.sunbirdTetherCounter <= 8) {
+          // Save tether
+          if (matches.source === 'Sunbird') {
+            for (const s of data.sunbirds) {
+              if (s.ID && s.ID.toString(16).toUpperCase() === matches.sourceId) {
+                const pos = Math.round(2 - 2 * Math.atan2(s.PosX - 100, s.PosY - 100) / Math.PI) % 4;
+                data.sunbirdTethers.push({ name: matches.target, pos: pos });
+              }
+            }
+          } else {
+            // If it's not from a sunbird it's p2p
+            data.p2pTethers.push({ source: matches.source, target: matches.target });
+          }
+          if (data.sunbirdTetherCounter < 8)
+            return;
+        }
+        // All 8 tethers collected, make the call
```

Comment:
What if somebody is dead?

Reply:
Good question. I think all 8 tethers always go out but no call would make real sense because you will die either way.
![grafik](https://user-images.githubusercontent.com/31262/150278068-d28fad3e-6572-4ba8-8de3-e4d3e2c7a086.png)
I _think_ this is the 8th tether retargeting because of a dead person unfortunately this would delay the call until the retargeting occured